test/unit/BCoWPool.t.sol

```diff
@@ -130,3 +148,104 @@ contract BCoWPool_Unit_EnableTrading is BaseCoWPoolTest {
     bCoWPool.enableTrading(appData);
   }
 }
+
+contract BCoWPool_Unit_Verify is BaseCoWPoolTest {
+  modifier assumeNotBoundToken(address _token) {
+    for (uint256 i = 0; i < TOKENS_AMOUNT; i++) {
+      vm.assume(tokens[i] != _token);
+    }
+    _;
+  }
+
+  function test_Revert_NonBoundBuyToken(address _otherToken) public assumeNotBoundToken(_otherToken) {
+    GPv2Order.Data memory order = correctOrder;
+    order.buyToken = IERC20(_otherToken);
+    vm.expectRevert(IBPool.BPool_TokenNotBound.selector);
+    bCoWPool.verify(order);
+  }
+
+  function test_Revert_NonBoundSellToken(address _otherToken) public assumeNotBoundToken(_otherToken) {
+    GPv2Order.Data memory order = correctOrder;
+    order.sellToken = IERC20(_otherToken);
+    vm.expectRevert(IBPool.BPool_TokenNotBound.selector);
+    bCoWPool.verify(order);
+  }
+
+  function test_Revert_LargeDurationOrder(uint256 _timeOffset) public {
+    _timeOffset = bound(_timeOffset, 5 minutes, type(uint16).max);
```

Comment:
inherit the constant

Reply:
wouldn't that be testing the implementation against itself?

Reply:
we're doing it like that on the other tests as well, such as `MIN_RATIO` or other things like that, in that way, if we change the constant the test will still pass

we don't have tests for "this constant value should be X"